pom.xml

```diff
@@ -44,7 +44,7 @@
         <sonar.organization>axoniq</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
-        <axonserver.api.version>2024.1.0</axonserver.api.version>
+        <axonserver.api.version>2025.1.0-SNAPSHOT</axonserver.api.version>
```

Comment:
I think we need to make sure that we construct a pull request for [this](https://github.com/AxonIQ/axon-server-api/tree/feature/dcb) Axon Server API before we can go ahead with this.

Reply:
We would need to merge it even to have a snapshot right?

Reply:
Yep, exactly!